Changes.md

```diff
@@ -1,3 +1,10 @@
+Unreleased
+---------------
+
+- Remove `Belt` as a dependency of `Stdlib`
```

Comment:
This is a breaking change. Most projects use Dune default config for implicit transitive deps, so they will have to add `melange.belt` to the list of `libraries` after upgrading. See https://github.com/melange-re/melange-re.github.io/pull/140.

Reply:
will mark it as such. thanks!